src/index.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-unused-vars */
+import _ from 'lodash';
+import './style.css';
+
+import { updateCompletedStatus, clearCompleted } from './modules/status.js';
+import addTask from './modules/Add.js';
+import deleteTask from './modules/delete.js';
+import { markAsComplete, markAsIncomplete } from './modules/statusupdate.js';
+
+const taskList = document.getElementById('task-list');
+const form = document.querySelector('form');
+
+let items = JSON.parse(localStorage.getItem('items')) || [];
+
+function renderTasks() {
+  taskList.innerHTML = '';
+  items.forEach((task) => {
+    const listItem = document.createElement('li');
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.checked = task.completed;
+    checkbox.addEventListener('change', () => {
+      updateCompletedStatus(task, items);
+      localStorage.setItem('items', JSON.stringify(items));
+      renderTasks();
+    });
```

Comment:
- [ ]  It would be better to use variables and [functions](https://github.com/Tracy-miranja/To-Do-peer-code-review/blob/eacee1822698b94182132f2ea0e6e75eb43962e2/src/index.js#L16-L27) after defining them.